examples/prometheus/rules/README.md

```diff
@@ -0,0 +1,31 @@
+# Prometheus and Alertmanager Rules
+
+## Loading Rules
+
+With this deployment method all files in the rules directory are mounted into the pod as a configmap.
+
+1. Create a configmap of the rules directory
+
+        oc create configmap base-rules --from-file=rules/
+1. Attach the configmap to the prometheus statefulset as a volume
+
+        oc volume statefulset/prometheus --add \
+           --configmap-name=base-rules --name=base-rules -t configmap \
+           --mount-path=/etc/prometheus/rules
+1. Delete pod to restart with new configuration
+
+        oc delete $(oc get pods -o name --selector='app=prometheus')
```

Comment:
do we want to say something about reload through sending HUP the prometheus process?

Reply:
This is a developer-focused workflow so I thought local files->restart was the most straight-forward path.

Reply:
Makes sense

Reply:
Maybe it's still worth to mention the HUP for operators out there. Up to you